cmd/juju/application/deployer/deployer.go

```diff
@@ -51,6 +51,8 @@ type localBundleDeployerKind struct {
 // localPreDeployerKind represents a local pre-deployed charm deployment
 type localPreDeployerKind struct {
 	userCharmURL *charm.URL
+	base         corebase.Base
+	imageStream  string
```

Comment:
It could be good to have a comments on the fields here. Mostly to make it clear what `imageStream` is here.

Reply:
Done